backend/pong/game_logic.py

```diff
@@ -0,0 +1,113 @@
+# pong/game_logic.py
+from .constants import *
+import math
+
+
+class PongGame:
+    def __init__(self):
+        # Initialize game state
+        self.player1_y = PLAYGROUND_HEIGHT // 2  # Initial position for player 1 (midpoint)
+        self.player2_x = PLAYGROUND_WIDTH - WALL_MARGIN
+        self.player1_x = WALL_MARGIN
+        self.player2_y = PLAYGROUND_HEIGHT // 2  # Initial position for player 2 (midpoint)
+        self.ball_x = PLAYGROUND_WIDTH - WALL_MARGIN - PLAYER_WIDTH   # Initial position for ball (x-coordinate)
```

Comment:
Not for this PR but for posterity, the ball should probably always start in the center with a random, but weighted probability distribution for its direction, kinda like a double funnel.